src/main/io/ledstrip.c

```diff
@@ -560,9 +589,35 @@ static void applyLedFixedLayers(void)
             const int auxInput = rcData[ledStripStatusModeConfig()->ledstrip_aux_channel];
             hOffset += scaleRange(auxInput, PWM_RANGE_MIN, PWM_RANGE_MAX, 0, HSV_HUE_MAX + 1);
         }
-
         color.h = (color.h + hOffset) % (HSV_HUE_MAX + 1);
-        setLedHsv(ledIndex, &color);
+
+        switch (fn) {
+#ifdef USE_GPS
+            case LED_FUNCTION_GPS_BAR:
+                if (ledBarStates[LED_BAR_GPS].count < gpsSol.numSat) {
```

Comment:
This will work quite poorly for low number  of GPS bar leds (<= 4) 
- color is already indicating state for low sat count (before fix)
- with fix, all leds are green, independent of number of sats

Maybe decide on 'reasonable' number of sats and interpolate to this number.

Reply:
the reasonable number is now gathered from the gps failsafe min sats, or defaulting at 8.
I would prefer it this way to have the association of "green" meaning "ready to fly", which is easy to understand for pilots.